src/main/java/wolox/training/repositories/BookRepository.java

```diff
@@ -2,7 +2,11 @@
 
```

Comment:
This shouldn't be here

Reply:
Please review again, this branch wasn't update with main.